### DISCUSSION TITLE
settings/configuration migration

### DISCUSSION THREAD

**Discussion by Masber** [keepassxreboot/keepassxc#11418] [Q&A] [answered]
Dear keepassxc community, 
I am installing keepassxc on my new machine,
I copied the password database over but I would also like to migrate my settings, specially because some password have been configured as OTP, also would like to move my tags over.
Is there a way to do this?

thank you

**Comment by droidmonkey** [comment, marked as answer]
TOTP and Tags are part of the database file.
